packages/api/tests/models/test-executions-model.js

```diff
@@ -183,6 +184,39 @@ test.serial('_getMutableFieldNames() returns correct fields for completed status
   t.deepEqual(updateFields, Object.keys(item));
 });
 
+test('storeExecutionRecord() can be used to create a new execution', async (t) => {
+  const { executionModel } = t.context;
+  const execution = fakeExecutionFactoryV2();
+  await executionModel.storeExecutionRecord(execution);
+
+  const fetchedItem = await executionModel.get({ arn: execution.arn });
+  t.is(fetchedItem.status, execution.status);
+});
+
+test('storeExecutionRecord() can be used to update an execution', async (t) => {
+  const { executionModel } = t.context;
+  const execution = fakeExecutionFactoryV2({
+    asyncOperationId: '1',
+    status: 'running',
+  });
+
+  await executionModel.storeExecutionRecord(execution);
```

Comment:
We should do a `.get` and assert the status/asyncOperationId being correct to ensure that the record is actually being updated